### PR TITLE
HL-202

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
[Link to ticket](url)

- Fixed the faucet FE in another commit
- This cleans up the utils

### CODE REVIEW

**Review by brettvniekerk — COMMENTED** [review]
This file could be moved into the formatting util file where shortenAddress is, and the test included in the related spec file

**Comment by bitcoinbrisbane** [comment]
> shortenHash could be moved into the formatting util file where shortenAddress is, and the test included in the related spec file

Agreed.  A default param could be passed too

**Review by OptimusOpus — APPROVED** [review]

File: src/utils/formatting.ts [hunk 1]
```diff
@@ -47,3 +47,12 @@ export const formatBetHistory = (
   blockNumber: +bet.createdAt,
   marketOracleResultSig: signedBetData.marketOracleResultSig
 });
+
+export const shortenAddress = (address: string) =>
+  `${address.slice(0, 5)}...${address.slice(address.length - 5)}`;
+
+export const shortenHash = (hash: string) => {
```
Comment (brettvniekerk) [inline src/utils/formatting.ts:54]: could be 
```js
export const shortenHash = (hash: string) => 
    `${hash.slice(0, 15)}...${hash.slice(hash.length - 15}`
```
but not a critical change.

**Review by brettvniekerk — APPROVED** [review]
One thing to consider otherwise LGTM

File: src/utils/formatting.ts (hunk 1 above)
Comment (bitcoinbrisbane) [reply src/utils/formatting.ts:54]: Can fix later.

**Pull request merged**